site.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# USAGE:
+#    site.sh              # serve website locally
+#    site.sh mybranch     # publish to https://jmfayard.github.io/rvdev/
+#    site.sh release      # publish to https://jmfayard.github.io/refreshVersions/
```

Comment:
The word "site" lacks explicitness about what it actually does.

I think this should be split into two scripts:
1. `build-website.sh`
2. `publish-website.sh`

The `publish-website.sh` script would:
1. Require a branch name to be passed as an argument.
2. Run the `build-website.sh` script.
3. Publish it at the right place.